tests/integration/test_digest_agent.py

```
@@ -0,0 +1,250 @@
+"""Integration tests for the DigestAgent.
+
+This module contains tests for the DigestAgent's functionality, including:
+- Unit tests for digest generation
+- Error handling tests
+- API endpoint tests (if implemented)
+"""
+
+import pytest
+import pytest_asyncio
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)